Libraries/PeriphDrivers/Source/SKBD/skbd_reva.c

```diff
@@ -119,11 +119,10 @@ int MXC_SKBD_RevA_Init(mxc_skbd_reva_regs_t *skbd, mxc_skbd_config_t config)
         }
 
         /* Configure the SKBD  */
-        temp =
-            ((config.reg_erase << MXC_F_SKBD_REVA_CTRL1_CLEAR_POS) | MXC_F_SKBD_REVA_CTRL1_AUTOEN);
-        temp |= (config.debounce << MXC_F_SKBD_REVA_CTRL1_DBTM_POS) & MXC_F_SKBD_REVA_CTRL1_DBTM;
-        temp |= (outputs << MXC_F_SKBD_REVA_CTRL1_OUTNB_POS) & MXC_F_SKBD_REVA_CTRL1_OUTNB;
-        skbd->ctrl1 |= temp;
+        skbd->ctrl1 = (config.reg_erase << MXC_F_SKBD_REVA_CTRL1_CLEAR_POS) | MXC_F_SKBD_REVA_CTRL1_AUTOEN // ->ctrl1[0]
+                	| MXC_F_SKBD_REVA_CTRL1_CLEAR // ->ctrl1[1]
+                	| MXC_F_SKBD_REVA_CTRL1_OUTNB //ctrl1[11:8]
+                	| (config.debounce << MXC_F_SKBD_REVA_CTRL1_DBTM_POS) & MXC_F_SKBD_REVA_CTRL1_DBTM; //ctrl1[15:13]
```

Comment:
Add another set of parenthesis around this again:
`((config.debounce << MXC_F_SKBD_REVA_CTRL1_DBTM_POS) & MXC_F_SKBD_REVA_CTRL1_DBTM)`

`MXC_F_SKBD_REVA_CTRL1_DBTM` will mask the entire register and not the specific debounce value.

Reply:
I unintentionally removed it before pushing 😩, fixed.